cmd/utils.go

```diff
@@ -437,50 +437,161 @@ func UserHomeDir() string {
 	return homeDir
 }
 
+func getConfigLabels(config *RootCommandConfig) (map[string]string, error) {
+	var labels = make(map[string]string)
+
+	projectConfig, projectConfigErr := getProjectConfig(config)
+	if projectConfigErr != nil {
+		return labels, projectConfigErr
+	}
+
+	labels["appsody.requested-stack"] = projectConfig.Stack
+
+	return labels, nil
+}
+
+func getGitLabels(config *RootCommandConfig) (map[string]string, error) {
+	gitInfo, err := GetGitInfo(config.Dryrun)
+	if err != nil {
+		return nil, err
+	}
+
+	var labels = make(map[string]string)
+
+	gitString := "git."
```

Comment:
These aren't going to all align, but lets use the OpenContainer annotations as much as possible.
https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys

Reply:
Do you mean using OpenContainer annotations for the git information? I haven't seen anything on that... Or do you mean adding OCI annotations for the final image being built?